app/controllers/admin/organizations_controller.rb

```diff
@@ -1,16 +1,17 @@
 class Admin::OrganizationsController < Admin::BaseController
+  include SortableTable
   before_filter :find_organization, only: [:show, :edit, :update, :destroy]
-  before_filter :set_sort_and_direction, only: [:index]
 
   def index
     page = params[:page] || 1
     per_page = params[:per_page] || 25
-    orgs = Organization.all
+    orgs = Organization.order(sort_column + " " + sort_direction).page(page).per(per_page)
```

Comment:
Apply pagination once, and as the final thing (where you're applying it lower is the correct place)